pdoc/test/__init__.py

```diff
@@ -1099,6 +1098,44 @@ def func(self):
         self.assertIsInstance(cls.doc['name'], pdoc.Variable)
         self.assertEqual(cls.doc['name'].type_annotation(), 'str')
 
+    def test_doc_comment_docstrings(self):
+        with temp_dir() as path:
+            filename = os.path.join(path, 'doc_comment_docstrs.py')
+            with open(filename, 'w') as f:
+                f.write('''#: Not included
+
+#: Line 1
+#: Line 2
+var1 = 1  #: Line 3
+
+#: Not included
+var2 = 1
+"""PEP-224 takes precedence"""
+
+#: This should not appear
+class C:
+    #: class var
+    class_var = 1
+
+    #: This also should not show
+    def __init__(self):
+       #: instance var
+       self.instance_var = 1
+''')
+
+            module = pdoc.import_module(filename)
+            pdoc_module = pdoc.Module(module)
+
+            cls_name_to_docs, instance_name_to_docs = pdoc._pep224_docstrings(pdoc_module)
+            self.assertEqual(cls_name_to_docs, {
+                                                'var2': 'PEP-224 takes precedence',
+                                                'var1': 'Line 1\nLine 2\nLine 3'})
+
+            pdoc_class = pdoc.Class('C', pdoc_module, module.C)
+            cls_name_to_docs, instance_name_to_docs = pdoc._pep224_docstrings(pdoc_class)
+            self.assertEqual(cls_name_to_docs, {'class_var': 'class var'})
+            self.assertEqual(instance_name_to_docs, {'instance_var': 'instance var'})
```

Comment:
```suggestion
            mod = pdoc.Module(filename)
            self.assertEqual(mod.doc['var1'].docstring, 'Line 1\nLine 2\nLine 3')
            self.assertEqual(mod.doc['var2'].docstring, 'PEP-224 takes precedence')
            self.assertEqual(mod.doc['C'].docstring, '')
            self.assertEqual(mod.doc['C'].doc['class_var'].docstring, 'class var')
            self.assertEqual(mod.doc['C'].doc['instance_var'].docstring, 'instance var')
```

Let's use public API to cover, not the implementation details. Ad lib, maybe this passes.

Reply:
Ok. Done.